### ISSUE TITLE
Impossible to escape when running `wp help` in subshell

### ISSUE THREAD

**Issue by elazzabi** [Automattic/vip-cli#284]
Currently, if you're in the sub shell and did run `wp help` you'll get a `less` interface with no possibility to escape (because we pause the input)

**Comment by elazzabi** [comment]
Been trying to fix this today. This happens because we `pause` the input when we run a command and then `resume` it when we receive the `end` event.

Even if we don't `pause` when we run `wp help`, any commands we run after the `wp help` should be considered as `less` commands and not `wp` commands. This is challenging as we need to keep two logics and will introduce more bugs.

I think we should intercept when help is run and just displays a message:
`For help running WP CLI commands, visit: https://wp-cli.org/`

cc @simonwheatley @nickdaugherty

**Comment by simonwheatley** [comment]
@mjangda, via P2:

----

When passing `--help`, you end up stuck in `more` (Edit: or `less`) 🙂 Any inputs (like spacebar) have no effect (so you can’t scroll or exit with a hard `ctrl+c`).

We could consider setting PAGER to `cat` to avoid passing stuff through to `more` at the container level (https://github.com/wp-cli/wp-cli/pull/2264) and instead let VIP CLI or the user handle it instead (probably the user for now).

But I’m assuming something isn’t working with our stdin support since there were also no response to a prompt despite repeated button mashing:

```
$ vip --app 103 -- wp user delete 8
✔ Which environment? · test-03
--reassign parameter not passed. All associated posts will be deleted. Proceed? [y/n] n

nasdfnsdf

# nothing
```

**Comment by nickdaugherty** [comment]
@simonwheatley is right that we're not hooking up the input stream properly (at all) right now.

**Comment by nickdaugherty** [comment]
@elazzabi I pushed some code to fix this in https://github.com/Automattic/vip/pull/300

BUT as you say, there's complication in our subshell because we need to distinguish between input sent to a command (prompts or `more`) and actual new commands.

For now I commented out the error when the line doesn't start with `wp` - that lets `help` work, but isn't perfect because you can still hit `Enter` multiple times in the readline interface and it just adds new spaces. You can also just type in nonsense and it doesn't do anything about it.

I think we'll need to change the behavior of the `line` event in our readline when a CLI command is currently in progress.

**Comment by nickdaugherty** [comment]
This was fixed by #300
